File: 598. Range Addition II/solution.js
/**
 * @param {number} m
 * @param {number} n
 * @param {number[][]} ops
 * @return {number}
 */
var maxCount = function(m, n, ops) {
  let maxRow = m;
  let maxCol = n;

  for (let i = 0; i < ops.length; i++) {
    maxRow = Math.min(maxRow, ops[i][0]);
    maxCol = Math.min(maxCol, ops[i][1]);
  }

  return maxRow * maxCol;
};

File: 598. Range Addition II/solution.go
func maxCount(m int, n int, ops [][]int) int {
	for _, v := range ops {
		m = min(m, v[0])
		n = min(n, v[1])
	}

	return m * n
}

func min(a, b int) int {
	if a < b {
		return a
	}

	return b
}